packages/foodie-backend/src/app.ts

```diff
@@ -3,9 +3,10 @@ import cookieParser from 'cookie-parser';
 import cors from 'cors';
```

Comment:
Is good for app file to be separate from the server file. Reference: https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/separateexpress.md

Reply:
app is currently separate from server